.travis.yml

```diff
@@ -16,9 +16,9 @@ script:
       [ "$TRAVIS_RUST_VERSION" = "1.20.0" ] || cargo test
     - cargo test --no-default-features
     - |
-      [ "$TRAVIS_RUST_VERSION" = "1.20.0" ] || cargo test --no-default-features --features "deflate-zlib"
-    - cargo test --no-default-features --features "deflate-miniz"
-    - cargo doc --no-deps --no-default-features --features "deflate-miniz,bzip2"
+      [ "$TRAVIS_RUST_VERSION" = "1.20.0" ] || cargo test --no-default-features --features "deflate"
```

Comment:
Can you check if it also works without the `[ "$TRAVIS_RUST_VERSION" = "1.20.0" ] || ` part? That was added because the previous implementation did not work on the 1.20.0 version anymore.

Reply:
The part `[ "$TRAVIS_RUST_VERSION" = "1.20.0" ] ||` is a check for the rust version. If the version matches, then it returns true without even running the test.
Since libflate seems to run fine on version 1.20.0, we do not need to skip the tests anymore. I think the travis script can become this:

```yaml
    - cargo test
    - cargo test --no-default-features
    - cargo doc --no-deps
    - rustdoc --test README.md -L target/debug
```

Reply:
great! I removed the additional checks.